e2e-tests/e2e-fixtures/dateConstants.js

```diff
@@ -60,6 +59,12 @@ export const twelveMonthsOneDayAgo = getFormattedValues(sub(todayDate, { months:
 export const oneYearAgo = getFormattedValues(sub(todayDate, { years: 1 }));
 export const twoYearsAgo = getFormattedValues(sub(todayDate, { years: 2 }));
 
+// Dates calculated from other values
```

Comment:
is this comment needed? seems out of place

Reply:
The other blocks of dates have comments `// Dates in the future` and `// Dates in the past` above them - I want to separate this date from the rest, since its calculated from `threeDaysAgo` and think this comment gets that across. Would be happy to change to something else though